code/modules/mob/living/carbon/carbon.dm

```diff
@@ -1,5 +1,4 @@
 /mob/living
-	var/canEnterVentWith = "/obj/item/implant=0&/obj/item/clothing/mask/facehugger=0&/obj/item/radio/borg=0&/obj/machinery/camera=0"
 	var/datum/middleClickOverride/middleClickOverride = null
```

Comment:
ok, I know this isnt your fault, but can you PLEASE move these other vars to `living_defines.dm` i HATE that we have these random defines slapped here in carbon.dm and it breaks VSC "go to definition" macros